### DISCUSSION TITLE
Sync Jumio Status?

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#130] [Q&A]
@maebeam @diamondhands0 is there a way that nodes can sync Jumio status for a user in a reliably way? 

Right now - the "Get your free $10" will still show even though user is verified because the users meta data is not on the node, as its handled by and stored on bitclout.com

**Comment by ItsAditya-xyz** [comment]
or better give an option to remove the "get free $10" ? tbh, a lot of users don't want it but still they are seeing it.

**Reply by tijno** [reply]
Yeah i thought about  it.

I can remove that easily on my own node so thats not the issue. But some nodes do want to show it but dont want to show it to users who already verified.

**Comment by derishaviar** [comment]
> @maebeam @diamondhands0 is there a way that nodes can sync Jumio status for a user in a reliably way?
> 
> Right now - the "Get your free $10" will still show even though user is verified because the users meta data is not on the node, as its handled by and stored on bitclout.com

Is there a way we could add somewhere on the profile if someone has done KYC? This would be quite helpful.

Also, I have a second account for a project I'm building. How could I KYC that account?
